project/game/riddlemaster_view.py

```diff
@@ -81,10 +81,10 @@ def on_click_open(self, event):
         if self.text.text.upper().strip() == "FIRE":
             next_view = VictoryView(self.scene, self.cast,self.script, self.props)
             self.window.show_view(next_view)
-        elif self.cast["lives"].get_text() > 0:
+        if self.cast["lives"].get_text() > 0:
```

Comment:
by changing these to 'if' instead of 'elif' the game_over is called if the wrong guess is made with one life left instead of zero.